hal/architecture/NRF5/MyHwNRF5.cpp

```diff
@@ -523,7 +523,18 @@ uint16_t hwCPUFrequency(void)
 
 int8_t hwCPUTemperature(void)
 {
-	return -127; // not implemented yet
+	uint32_t Temperature = 0;
```

Comment:
Should this be signed? I guess we could get negative temperatures?

Reply:
I fix Temperature type same as NRF_TEMP->TEMP type int32_t